src/status_im/ui/screens/accounts/events.cljs

```diff
@@ -165,8 +165,10 @@
   returns all effects necessary for account update."
   [{{:keys          [network]
      :accounts/keys [accounts current-account-id] :as db} :db now :now} new-account-fields]
-  (let [current-account (get accounts current-account-id) 
-        new-account (update-account current-account new-account-fields now)]
+  (let [current-account (get accounts current-account-id)
+        ;;TODO(rasom): decide if we really need to use :now coefx here
```

Comment:
I understand the concern here about proliferation of `(inject-cofx :now)` in every handler which eventually calls `account-update`.
However, I still think it's very wise to keep the function totally pure, for example it would be beneficial in unit testing when comparing return values. 
What I propose is to just include `(inject-cofx :now)` (and any other widely used coeffects) in our standard interceptor chain, that way, current timestamp will be always available for any event handler.

Reply:
done